### PR TITLE
translate tutorials/kubernetes-basics/update/update-intro/ in Italian

### PR DESCRIPTION
**Pull request by micheleberardi**
translate tutorials/kubernetes-basics/update/update-intro/ in Italian

### CODE REVIEW

**Comment by rlenferink** [comment]
@micheleberardi when you think this is ready to be merged, just give us a ping !

**Comment by micheleberardi** [comment]
Hi @rlenferink 
how are you ?
to translate 100% it will take more time but  if today i check the other's language are not ready 100% i try to catch  the same point for the end of the next week 
let me know if is ok
thanks have a great day

File: content/it/case-studies/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Owned by Kubernetes Blog reviewers.
```
Comment (rlenferink) [inline content/it/case-studies/OWNERS]: Can you remove this file? This is not needed right now and can be easily added at a later point.

File: content/it/blog/OWNERS [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Owned by Kubernetes Blog reviewers.
```
Comment (rlenferink) [inline content/it/blog/OWNERS]: Can you remove this file? This is not needed right now and can be easily added at a later point.

File: content/it/docs/concepts/overview/what-is-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
+---
+reviewers:
+- bgrant0607
+- mikedanese
```
Comment (rlenferink) [inline content/it/docs/concepts/overview/what-is-kubernetes.md]: Please remove the 

```bash
reviewers:
- bgrant0607
- mikedanese
```

part from this file

Comment (micheleberardi) [reply content/it/docs/concepts/overview/what-is-kubernetes.md]: Hi :)
I think i removed everything that you ask
Thanks

Comment (rlenferink) [reply content/it/docs/concepts/overview/what-is-kubernetes.md]: @micheleberardi thanks. you updated this file correctly. However, can you please remove the following files as well:
```bash
content/it/case-studies/OWNERS
content/it/blog/OWNERS
```

File: i18n/it.toml [hunk 4]
```diff
@@ -0,0 +1,82 @@
+# i18n strings for the English (main) site.
```
Comment (rlenferink) [inline i18n/it.toml]:
```suggestion
# i18n strings for the Italian site.
```

File: i18n/it.toml [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+other = "Sottoscrivi"
+
+[main_contribute]
+other = "Contribuire"
+
+[main_edit_this_page]
+other = "Modifica questa pagina"
+
+[main_page_history]
+other ="Pagina History"
```
Comment (rlenferink) [inline i18n/it.toml]:
```suggestion
other = "Pagina History"
```

File: content/it/_index.html [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+---
+
+{{< deprecationwarning >}}
+
+{{< blocks/section id="oceanNodes" >}}
+{{% blocks/feature image="flower" %}}
+### [Kubernetes (k8s)]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}})
+
+è un sistema open source per automatizzare la distribuzione il ridimensionamento e la gestione di applicazioni containerizzate.
+{{% /blocks/feature %}}
```
Comment (rlenferink) [inline content/it/_index.html]:
```suggestion
```

Please remove this line as it breaks the build by closing 1 opening tag twice

**Comment by rlenferink** [comment]
/hold cancel
/lgtm

@zacharysarah for approval

**Comment by rlenferink** [comment]
Looks like `/cc` doesn't accept an additional sentence 🙊

**Comment by rlenferink** [comment]
Testing: 
/language it

**Comment by rlenferink** [comment]
@lledru wants to help out as well

File: OWNERS_ALIASES [hunk 7]
```diff
@@ -161,6 +161,12 @@ aliases:
     - erickhun
     - jygastaud
     - awkif
+  sig-docs-it-owners: #Team: Italian docs localization; GH: sig-docs-it-owners
```
Comment (zacharysarah) [inline OWNERS_ALIASES]: @micheleberardi Please add yourself to these aliases as well. Before this PR can merge, you must first [open an issue](https://github.com/kubernetes/org/issues) to join the Kubernetes GitHub org and be approved to join. I'm happy to sponsor you; @rlenferink 👋 are you willing to co-sponsor?

Comment (micheleberardi) [reply OWNERS_ALIASES]: Hi guys how are u? 
I’m traveling in west cost these days I will back in NYC Thursday night 
I can do that when I back 
It’s ok or too late ? 
Thanks

Comment (micheleberardi) [reply OWNERS_ALIASES]: I did all steps above 
Thanks for the support

Comment (rlenferink) [reply OWNERS_ALIASES]: Thanks for the review @zacharysarah 
I am willing to co-sponsor @micheleberardi

**Comment by zacharysarah** [comment]
@micheleberardi @rlenferink I think @parispittman would like to send you some contributor swag for getting this localization started! ✨ Please reach out to her with your email addresses.

**Comment by micheleberardi** [comment]
HI @zacharysarah  @rlenferink @parispittman 
sure i will
Regards

**Pull request merged**